### PR TITLE
Register Subdomain: zoro.xen

### PR DESCRIPTION
**Pull request by xenmods**
Added the domain: zoro.xen

### CODE REVIEW

File: domains/zoro.xen.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "xenmods",
+        "email": "ilumomin04@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "db112aae-90b3-4d1b-a302-e4da8d2cae38.id.repl.co"
```
Comment (WilliamDavidHarrison) [inline domains/zoro.xen.json:9]: I can't seem to reach this website.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/zoro.xen.json (hunk 1 above)
Comment (xenmods) [reply domains/zoro.xen.json:9]: it is up now.

**Comment by xenmods** [comment]
the website is up now

File: domains/zoro.xen.json (hunk 1 above)
Comment (WilliamDavidHarrison) [reply domains/zoro.xen.json:9]: @xenmods What ezactly does this website do?

**Review by wdhdev — APPROVED** [review]

**Review by TheBotlyNoob — CHANGES_REQUESTED** [review]
the repl doesn't work

**Comment by TheBotlyNoob** [comment]
I'm closing this because of the large amounts of pull requests. You can make another PR later with fixes if you want.

**Pull request closed without merging**